pkg/traceql/parse_test.go

```diff
@@ -924,8 +923,7 @@ func TestSpansetFilterOperators(t *testing.T) {
 		{in: "{ .a = true }", expected: newBinaryOperation(OpEqual, NewAttribute("a"), NewStaticBool(true)), alsoTestWithoutSpace: true},
```

Comment:
let's also add test case for `NIL NEQ NIL` and  `NIL EQL NIL`?

Reply:
done